api/v1beta1/foundationdb_version.go

```diff
@@ -155,3 +155,29 @@ func (version FdbVersion) HasSidecarCrashOnEmpty() bool {
 func (version FdbVersion) HasNonBlockingExcludes() bool {
 	return version.IsAtLeast(FdbVersion{Major: 6, Minor: 3, Patch: 5})
 }
+
+// Versions provides a shorthand for known versions.
+// This is only to be used in testing.
+var Versions = struct {
```

Comment:
If this is only intended for testing purpose we should move it into the `internal` package and not expose it to the outside.

Reply:
I tried that but couldn't get the imports to work, because it created a circular import.